services/shhext/chat/encryption_test.go

```diff
@@ -749,30 +773,33 @@ func (s *EncryptionServiceTestSuite) TestBundleNotExisting() {
 // A new bundle has been received
 func (s *EncryptionServiceTestSuite) TestRefreshedBundle() {
 
+	config := DefaultEncryptionServiceConfig("none")
+	// Set up refresh interval to "always"
+	config.BundleRefreshInterval = 1000
+
+	s.initDatabases(&config)
+
 	bobKey, err := crypto.GenerateKey()
 	s.Require().NoError(err)
 
 	aliceKey, err := crypto.GenerateKey()
 	s.Require().NoError(err)
 
 	// Create bundles
-	bobBundle1, err := NewBundleContainer(bobKey, bobInstallationID)
+	bobBundle1, err := s.bob.CreateBundle(bobKey)
 	s.Require().NoError(err)
+	s.Require().Equal(uint32(1), bobBundle1.GetSignedPreKeys()[bobInstallationID].GetVersion())
 
-	err = SignBundle(bobKey, bobBundle1)
-	s.Require().NoError(err)
+	// Sleep the required time so that bundle is refreshed
+	time.Sleep(1000 * time.Millisecond)
```

Comment:
```suggestion
	time.Sleep(config.BundleRefreshInterval * time.Millisecond)
```

Reply:
This would express more clearly the intent and would be less subject to be broken.

Reply:
done, thanks!